### PR TITLE
Dani/feature/recipe tests

### PR DESCRIPTION
**Pull request by daniabee**
This PR adds the commits for the tests, sample data, and code for the recipe class. 
The data used for testing can be found in the sampleSet file recipe file. 
Methods where dynamically created and can be adjusted to better fit other classes if need be. 

If there are any adjustments that need to be made to account for the other classes, please let me know!

### CODE REVIEW

File: test/Recipe-test.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
+import { expect } from "chai";
+import ingredientsData from "../src/data/ingredients";
+import recipeSampleData from "../src/sampleSets/recipeSample";
+import Recipe from "../src/classes/Recipe";
+
+describe("Recipe", function () {
+  let recipe1, recipe2, recipe3;
```
Comment (AndrewCerveny) [inline test/Recipe-test.js:7]: I like this never heard or seen anything like this... interesting way to declare variables.

**Review by AndrewCerveny — COMMENTED** [review]
<7> A very innovative way to declare variables. 
<10> can you explain to me - what the data set refers to. was it the <whole ingridents list itself?>
<11> I think it's a style thing. I didn't know return as a keyword can be used like that. I've always declared a variable  @ the top and then lower outside the inner execution I place a return, then an outer return to reference the top variable.  It's streamlined I like it. 

Can we change  get cost in cents ~ to get cost in dollars. ..   

Any questions reach out!

**Review by AndrewCerveny — COMMENTED** [review]
<30> when you have time can you explain to me why you created a new ingrediantObj

**Comment by daniabee** [comment]
> <30> when you have time can you explain to me why you created a new ingrediantObj

This was actually a redundant bit of code from a different output I was playing around with! It shouldn't be there! Thanks for the second pair of eyes!

**Comment by daniabee** [comment]
> <7> A very innovative way to declare variables. <10> can you explain to me - what the data set refers to. was it the <whole ingridents list itself?> <11> I think it's a style thing. I didn't know return as a keyword can be used like that. I've always declared a variable @ the top and then lower outside the inner execution I place a return, then an outer return to reference the top variable. It's streamlined I like it.
> 
> Can we change get cost in cents ~ to get cost in dollars. ..
> 
> Any questions reach out!

Method for cost has been updated to return a dollar amount as requested!

**Pull request merged**